src/parsers/parsers.ts

```diff
@@ -175,6 +176,8 @@ export const parsers: Parser[] = [
   new DMOJProblemParser(),
   new DMOJContestParser(),
 
+  new DimikOJProblemParser(),
```

Comment:
Please move this line up to in-between the `CSUACMOnlineJudgeContestParser` and `DMOJProblemParser` constructor calls to keep the array sorted alphabetically.